### DISCUSSION TITLE
grafana v8.x mixed don't support variable datasource

### DISCUSSION THREAD

**Discussion by xuhrc** [grafana/grafana#42913] [Data Sources]
grafana v8.3.0
I defined a variable datasource called {prometheus}
![image](https://user-images.githubusercontent.com/5369728/145378488-04524c44-93d3-436d-a37f-4259a0abc83e.png)

when set mixed datasource, the query datasource filtered the dyanamic datasource defined in the variables.
![image](https://user-images.githubusercontent.com/5369728/145378660-788f0aba-2a3a-4d13-a840-d8ddf4c678dc.png)

**Comment by joshzcold** [comment]
Is there any traction on this? 
Seems like something that is pretty obviously missing.
im on version `v8.4.5`

**Reply by joshzcold** [reply]
Looks like it works in `9.0.7` so ill be looking to upgrade

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
